app/views/lists/show.html.erb

```diff
@@ -10,5 +10,26 @@
   <%= @list.description %>
 </p>
 
+<ul>
+  <% @list.papers.each do |paper| %>
+    <li>
+      <%=paper.title%>
+    </li>
```

Comment:
Really minor style comment, but this doesn't need to be nested (on different lines)